ansible/cluster.yml

```diff
@@ -44,6 +44,12 @@
   tags:
     - network-service-install
 
+# install contiv netmaster
```

Comment:
I believe we can remove the redundant hosts entries for each networking provider in cluster.yml, and use ```when: networking == flannel | contrail | contiv``` under the appropriate hosts entries to select the proper networking role. I was just trying to follow existing patterns with the contiv PR. Could this PR be accepted as-is? I can create an issue such as "improve support for multiple networking providers" and submit a follow-on PR that addresses the issue.